.circleci/config.yml

```diff
@@ -1116,9 +1116,9 @@ workflows:
           filters:
             branches:
               only:
-                # - master
+                - master
                 - graphqlschemae2e
-                # - beta
+                - beta
```

Comment:
Is this related to the mock changes?

Reply:
No , this is because of issues in circleCI yesterday. I am now changing it back to original